process/block/baseProcess.go

```diff
@@ -2110,3 +2115,19 @@ func (bp *baseProcessor) setNonceOfFirstCommittedBlock(nonce uint64) {
 	bp.nonceOfFirstCommittedBlock.HasValue = true
 	bp.nonceOfFirstCommittedBlock.Value = nonce
 }
+
+func (bp *baseProcessor) checkSentSignaturesBeforeCommitting(header data.HeaderHandler) error {
```

Comment:
`checkSentSignaturesAtCommitTime` since it is called in the CommitBlock function, not before

Reply:
fixed